examples/02_meta-analyses/10_plot_cbmr.py

```diff
@@ -0,0 +1,362 @@
+"""
+
+.. _metas_cbmr:
+
+===========================================
+Coordinate-based meta-regression algorithms
+===========================================
+
+A tour of CBMR algorithms in NiMARE
+
+This tutorial is intended to provide a brief description and example of the CBMR
+algorithm implemented in NiMARE. For a more detailed introduction to the elements
+of a coordinate-based meta-regression, see other stuff.
+"""
+import numpy as np
+import scipy
+from nilearn.plotting import plot_stat_map
+
+from nimare.generate import create_coordinate_dataset
+from nimare.meta import models
+from nimare.transforms import StandardizeField
+
+###############################################################################
+# Load Dataset
+# -----------------------------------------------------------------------------
+
```

Comment:
Explain what you're going to load and why.

For example:
"Here, we're going to simulate a dataset that.... " and list the properties of the simulated dataset.

Reply:
Agree :) I've added a detailed explanation of how the dataset is simulated